staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -379,6 +384,10 @@ type JobStatus struct {
 	// +optional
 	Failed int32 `json:"failed,omitempty" protobuf:"varint,6,opt,name=failed"`
 
+	// The number of pods which are terminating.
+	// +optional
+	Terminating int32 `json:"terminating,omitempty" protobuf:"varint,10,opt,name=terminating"`
```

Comment:
this being a non-pointer is likely where your proto serialization change is coming from

Reply:
That worked!